.github/workflows/common.yml

```diff
@@ -31,6 +33,8 @@ jobs:
     name: Unit test
     runs-on: ubuntu-18.04
     timeout-minutes: 15
+    env:
+      RABBITMQ_HOST: localhost
```

Comment:
This is not a common environment variable.

Reply:
Fixed.